SDKGenerator.njsproj

```diff
@@ -101,6 +101,11 @@
     <OutputPath>bin\</OutputPath>
     <ScriptArguments>-destpath ..\sdks\XPlatCppSdk -apiSpecGitUrl -buildIdentifier XPlatCppSdk_manual</ScriptArguments>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'XPlatCSdk' ">
+    <DebugSymbols>true</DebugSymbols>
+    <OutputPath>bin\XPlatCSdk\</OutputPath>
+    <ScriptArguments>-destpath ..\..\src\playfabcorec\playfabcorecsdk -apiSpecGitUrl -buildIdentifier playfabcorecsdk_manual</ScriptArguments>
```

Comment:
>playfabcorecsdk_manual [](http://example.com/codeflow?start=102&length=22)

Similarly I think this should be PlayFabCoreCSdk_manual because I find it to be more readble